src/greeter/waylandkeyboardbackend.cpp

```diff
@@ -59,7 +108,10 @@ void WaylandKeyboardBackend::dispatchEvents()
 
 void WaylandKeyboardBackend::connectEventsDispatcher(KeyboardModel *model)
 {
-    Q_UNUSED(model);
+    connect(model, &KeyboardModel::currentLayoutChanged, this, [this, model] {
```

Comment:
This has nothing to do with connecting event dispatchers. 

This is more akin to Backend::sendChanges which tells backends to sync chosen items out.
Changing also probably means the QObject inheritance is not needed.